Update emojis.md (#985)
Bedrock-OSS/bedrock-wiki: docs/concepts/emojis.md

-   To position an emoji towards the left or right, simply add any pixel of 5-10% opacity to its side (within its own slot), opposite to the direction you want to move it.
    -   Ex: to move an emoji to the right by 2 pixels, add any 5-10% opacity pixel connecting to it, anywhere on its left-most side and add another one to the left of that pixel.

> _Note: the following steps are only for emojis that do not fit to its slot width (smaller than its slot size.)_

Reviewer: ```suggestion
_Note: The following steps apply only to emojis that do not fill their slot width (i.e., are smaller than the slot size)._
```

### Glyph Separation Space

Sometimes, it appears that if you put 2 glyphs near to each other, there will be a couple of empty pixels between them. The only fix for it is to scale the glyph (png file) itself. Ex: 256×256 → 512×512

### Resizing the Emoji

> To resize the emoji, change the resolution of the glyph (png file) to 512×512, the reason you should double the resolution is because then you will have doubled the amount of pixels, and so therefore, more room for editing, Once you have done this, in a image editor program of your choise just select the new emoji you have created, and size it down and center it in the middle of the emoji slot, it should then give your emoji more space around it when you use the character associated with it in the game.

Reviewer: ```suggestion
To resize the emoji, change the resolution of the glyph (PNG file) to 512×512. The reason for doubling the resolution is that it increases the number of pixels, providing more room for editing. Once you've done this, open an image editor of your choice, select the new emoji, resize it, and center it within the emoji slot. This will give the emoji more space around it when used in the game.
```

### Understanding the Hex System

> The glyph (png file)'s name contains the first part of the hex code for the character, for example, the glyph's name could be `glyph_E1`, we are using `E1` because it has more space,

Reviewer: ```suggestion
The glyph (PNG file) name contains the first part of the hex code for the character. For example, the glyph's name could be `glyph_E1`. We use `E1` because it has more space.
```

> you can download this and use it as a guide to make sure your emoji doesnt come outside of its own designated emoji slot,

Reviewer: ```suggestion
You can download this and use it as a guide to make sure your emoji does not extend beyond its own designated emoji slot.
```

<WikiImage
    src="/assets/images/concepts/emojis/custom/annotated/glyph_E1.png"
    caption="RP/font/glyph_E1.png"
    width="512"
    pixelated
/>

> As you can see in the image, the rows and columns have numbers and letters, the numbers correspond with the second part of the hex character code, you can use the unicode character code point to make it easyer to understand, lets say we are adding something at `10`.

Reviewer: ```suggestion
As you can see in the image, the rows and columns have numbers and letters. The numbers correspond to the second part of the hex character code. You can use the unicode character code point to make it easier to understand. Let's say we are adding something at `10`.
```

> The unicode character code point for this would simply be `U+E110`, using this translater that i have slightly modified, you can use it to get the actual character to copy and paste into minecraft to see the emoji at the designated slot in the png file.

Reviewer: ```suggestion
The unicode character code point for this would simply be `U+E110`. Using this translator, which I have slightly modified, you can obtain the actual character to copy and paste into Minecraft to see the emoji in its designated slot in the PNG file.
```

> If your using development_resource_packs you can use `/reload all` to easily reload the behaviour packs and the resource packs you have in the minecraft world.

Reviewer: ```suggestion
If you're using `development_resource_packs`, you can use `/reload all` to easily reload the behaviour packs and resource packs in your Minecraft world.
```
